messages/4.2.0.txt

```diff
@@ -0,0 +1,10 @@
+SublimeLinter-eslint 4.2.0
+--------------------------
+
+We can *fix* that!  We have two new settings:
+
+- "filter_fixables" (true|false): If set to true, the plugin will ignore
+  errors/warnings which are fixable.
+
+- "fix_on_save" (true|false): If set to true, will attempt to auto-fix
+  the source file. This is for pure '.js' files only atm.
```

Comment:
Oh, as in not for embedded code? Does it even work for embedded code outside SL?

Reply:
Yes, no embedded code, only files. Maybe it can fix vue files or json or ... but who knows that?

Reply:
> but who knows  

I certainly don't, I just dabble in JS.